Sources/PrimerSDK/Classes/Core/Crypto/AES256.swift

```diff
@@ -0,0 +1,172 @@
+//
```

Comment:
Even though I'm not encrypting analytics event anymore, maybe we'll need to encrypt/decrypt data in the future